examples/js/loaders/GLTF2Loader.js

```diff
@@ -2652,6 +2652,11 @@ THREE.GLTF2Loader = ( function () {
 
 							}
 
+							//do not clone children as they will be replaced anyway
+							var children = group.children
+							group.children=[];
+							var clonedgroup = group.clone();
+							group.children=children;
```

Comment:
I need to insert the group and can't overwrite the _node

_node           = gltf.node
clonedgroup = gltf.mesh
child              = gltf.primitive

but of course I can use the nonrecursive clone function instead of writing my own ..! ( sorry )